examples/ola-recorder.cpp

```diff
@@ -154,6 +170,41 @@ int VerifyShow(const string &filename) {
   }
 }
 
+
+/**
+ * Playback a recorded show
+ */
+int PlaybackShow() {
+  const string filename = FLAGS_playback.str();
+  if (FLAGS_verify_playback) {
+    // Verify the show and print a summary before running
+    string summary;
+    const int verified = VerifyShow(filename, &summary);
+    // Printing a newline first makes this output look better in interactive
+    // terminal logs.
+    OLA_INFO << endl << summary;
+    if (verified != ola::EXIT_OK) {
+      // Show did not pass verification
+      // The Verify method has already informed the user, so fail with that
+      // method's result.
```

Comment:
Because we've merged the output, and force it down to info level, at the default warn level we won't get any output, it will just fail. It would be good to either get an error message here, or just log a generic verify failed message.

Reply:
The summary info is at info level, but there are still log messages from ShowPlayer that get sent to fatal, error, or warn that will have been shown.

After mashing my keyboard into the show file to make it invalid, I get:
```
ola_recorder --verify test.show
examples/ShowLoader.cpp:139: Line 15 invalid: sergtesdrgtsd
examples/ShowPlayer.cpp:349: Invalid data at line 15
```